scalability_and_performance/cluster-compare/understanding-the-cluster-compare-plugin.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="understanding-the-cluster-compare-plugin"]
+= Understanding the cluster-compare plugin
+include::_attributes/common-attributes.adoc[]
+:context: understanding-cluster-compare
+
+toc::[]
+
+The `cluster-compare` plugin compares a cluster configuration with a reference configuration. The plugin reports configuration differences while suppressing expected variations by using configurable validation rules and templates. 
```

Comment:
I'm reviewing this from the perspective of someone that isn't really familiar with this tool - suggest explicitly stating that this is an `oc` cli plugin
```suggestion
The `cluster-compare` tool is an OpenShift CLI (`oc`) plugin that compares a cluster configuration with a reference configuration. The plugin reports configuration differences while suppressing expected variations by using configurable validation rules and templates. 
```